<launch>

    <arg name="ns" default="/" />
    <group ns="$(arg ns)">
        <node name="visualization_node" pkg="bobi_vision" type="visualization_node" output="screen"/>
    </group>

</launch>